.github/workflows/ci-cd.yaml

```diff
@@ -50,3 +50,21 @@ jobs:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           publish_dir: ./build
         if: github.ref == 'refs/heads/master'
+      - name: Publish to Netlify draft URL
+        uses: netlify/actions/cli@3e1877e3a4afe02687a10b7fa7f2456bf35d1925
+        with:
+          args: deploy --dir=./build
```

Comment:
Maybe `args: ${{ github.ref == 'refs/heads/master' }} && deploy --prod --dir=./build || deploy --dir=./build` would work but I thought it's too obscure.